### PR TITLE
Log executor only if logging is enabled

### PR DESCRIPTION
**Pull request by kornys**
### Type of change

- Enhancement

### Description

Logging of running commands cause really mess in our logs, so I changed that behavior to log that command only if logging is enabled.

### CODE REVIEW

File: systemtests/src/main/java/io/enmasse/systemtest/executor/Exec.java [hunk 1]
```diff
@@ -35,7 +35,7 @@
  * Class provide execution of external command
  */
 public class Exec {
-    private static Logger log = LoggerUtils.getLogger();
+    private static final Logger LOGGER = LoggerUtils.getLogger();
```
Comment (ctron) [inline systemtests/src/main/java/io/enmasse/systemtest/executor/Exec.java:38]: I would prefer to keep this `log` as we have it in all the other classes as well.

Comment (kornys) [reply systemtests/src/main/java/io/enmasse/systemtest/executor/Exec.java:38]: in many classes it was already renamed to a LOGGER, so if I see class with log I try ti refactor that

File: systemtests/src/main/java/io/enmasse/systemtest/platform/Kubernetes.java [hunk 2]
```diff
@@ -173,11 +173,11 @@ public static Kubernetes getInstance() {
             isOpenshift = Exec.execute(Arrays.asList(Kubernetes.getInstance().getCluster().getKubeCmd(), "api-resources"), false).getStdOut().contains("openshift.io");
             ocpVersion = instance.getOcpVersion();
 
-            log.info("Is CRC: {}", isCRC());
-            log.info("Is OpenShift: {}", isOpenShift());
-            log.info("Is OpenShiftCompatible: {}", isOpenShiftCompatible());
+            log.debug("Is CRC: {}", isCRC());
```
Comment (ctron) [inline systemtests/src/main/java/io/enmasse/systemtest/platform/Kubernetes.java:176]: Because down here it is still `log`

Comment (kornys) [reply systemtests/src/main/java/io/enmasse/systemtest/platform/Kubernetes.java:176]: yeah it is a different class :) ok I will move back to a log

**Pull request merged**